handlers.go

```diff
@@ -87,6 +87,11 @@ func signupSubmit(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("HX-Redirect", "/dashboard")
 }
 
+func logoutSubmit(w http.ResponseWriter, r *http.Request) {
```

Comment:
This seems to have become a helper function that just logs a user out. We should rename ti to logout(), bc thats what it does, it logs out